aicsimageio/readers/czi_reader.py

```diff
@@ -132,7 +132,28 @@ def scenes(self) -> Tuple[str, ...]:
                 czi = CziFile(open_resource)
                 xpath_str = "./Metadata/Information/Image/Dimensions/S/Scenes/Scene"
                 meta_scenes = czi.meta.findall(xpath_str)
-                scene_names = [x.get("Name") for x in meta_scenes]
+                scene_names: List[str] = []
+
+                # Some "scenes" may have the same name but each scene has a sub-scene
+                # "Shape" with a name.
+                #
+                # An example of this is where someone images a 96 well plate with each
+                # well being it's own scene but they name every scene the same value.
+                # The sub-scene "Shape" elements have actual names of each well.
+                #
+                # If we didn't do this, the produced list would have 96 of the same
+                # string name making it impossible to switch scenes.
```

Comment:
this can happen with ome-tiff also, no?  non-unique scene names?

Reply:
I sense a `set_scene_index` coming...

Reply:
I mean it's possible..... We use the "Image:ID" for OME-TIFF which would be super weird to have multiple images with the same ID and I would argue, be breaking the spec.